src/lib/stores/resourceEditor.ts

```diff
@@ -0,0 +1,7 @@
+import { writable } from 'svelte/store';
+
+export const isLeftToRight = writable<boolean>((localStorage.getItem('isLeftToRight') || 'true') === 'true');
```

Comment:
Naming nit: maybe something like `editPaneOnLeft` would be more self explanatory?